src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -1700,7 +1700,7 @@ class TopToolbar final : public Window
 
         auto screenPos = sourceScreenPos;
         uint16_t maxPossibleHeight = ZoomLevel::max().ApplyTo(
-            std::numeric_limits<decltype(TileElement::BaseHeight)>::max() - 32);
+            std::numeric_limits<decltype(TileElement::BaseHeight)>::max() - 1);
```

Comment:
This is unrelated to the fix for stuff getting cut off. In fact, this makes the problems more likely to occur.

Reply:
> This is unrelated to the fix for stuff getting cut off. In fact, this makes the problems more likely to occur.

Yeah, that's what I mentioned in the discord, I'm not sure why this even showed up in here. That's from a completely different PR, where I fixed being able to place walls and small scenery higher up.
What about it makes problems more likely, though? I've also been using that change locally, and it's worked wonderfully.